### ISSUE TITLE
HeaderView is hidden at first run of the tableView.

### ISSUE THREAD

**Issue by glennposadas** [gskbyte/GSKStretchyHeaderView#83]
Hi, I've been using this awesome library since last year and this is my first time to have an encounter with an issue. Basically the issue is that the headerview is hidden at the first run of the screen / tableView. See photos below. 
More info: 
1. I need to scroll up (or down?) to show the strechy header view.
2. This issue happens if the tableView row count exceeds at I think 10, if I have like 1 to 10 rows only, I get the expected result.

**EXPECTATION**

![screen shot 2018-01-06 at 12 23 31 am](https://user-images.githubusercontent.com/12502679/34618317-539596da-f279-11e7-8daa-f9e77ac65a18.png)

* *CURRENT RESULT**
![screen shot 2018-01-06 at 12 22 00 am](https://user-images.githubusercontent.com/12502679/34618324-584842a4-f279-11e7-92ad-98c4f532f0f9.png)

Is this normal? How to get the expected result? :) Thank you!

Codes:

**ViewController**

    private lazy var homeHeaderView: GSKStretchyHeaderView = {
        let view = HomeHeaderView(frame: UIScreen.main.bounds)
        view.minimumContentHeight = 0
        view.maximumContentHeight = 80.0
        view.contentShrinks = false // does not matter
        view.expansionMode = .topOnly
        return view
    }()

    override func viewDidLoad() {
        super.viewDidLoad()
        
        // Add subviews
        
        self.view.addSubview(self.tableView)
        self.tableView.snp.makeConstraints { (make) in
            make.top.equalTo(self.view.snp.topMargin)
            make.leading.trailing.bottom.equalToSuperview()
        }
        
        self.tableView.addSubview(self.homeHeaderView)
        if #available(iOS 11.0, *) {
            self.tableView.contentInsetAdjustmentBehavior = .never
        }
    }

**HeaderView**

    override init(frame: CGRect) {
        super.init(frame: frame)
        
        self.backgroundColor = .white
        
        // Setup header subviews.
        
        self.addSubview(self.imageView_Logo)
        self.imageView_Logo.snp.makeConstraints { (make) in
            make.width.equalTo(134.0)
            make.height.equalTo(80.0)
            make.top.leading.trailing.equalToSuperview()
            make.bottom.equalToSuperview().inset(35.0)
        }
    }

  EDIT: I think this is normal, I tried experimenting by changing the number of rows of my tableview. But is there a way though to get the expected result regardless of the number of rows? :D Thank you!

**Comment by gskbyte** [comment]
It looks like this is because of my good friend iOS 11. If that's the case, this might work:

```swift
if #available(iOS 11.0, *) {
    self.tableView.contentInsetAdjustmentBehavior = .never
}
```

**Comment by glennposadas** [comment]
Okay, thank you for that. As you can see, I already have such piece of code above. :) This issue is nice to have though, but hopefully we can come up with a better solution in the future.
